server/service/src/purchase_order_line/insert/mod.rs

```diff
@@ -51,24 +53,17 @@ pub fn insert_purchase_order_line(
             let validate_input = validate::ValidateInput {
                 id: input.id.clone(),
                 purchase_order_id: input.purchase_order_id.clone(),
-                item_id: input.item_id.clone(),
+                item_id: Some(input.item_id.clone()),
+                item_code: None,
                 // TODO amend default value if we extend standard insert line input
                 requested_pack_size: input.requested_pack_size.unwrap_or_default(), // Default value
             };
             validate(&ctx.store_id.clone(), &validate_input, connection)?;
-
-            activity_log_entry(
```

Comment:
side quest.. activity log is being logged twice and this isn't even saving the correct id..